app/Jobs/ImportTurboVotePosts.php

```diff
@@ -33,6 +33,17 @@ public function __construct($filepath)
         $this->filepath = $filepath;
     }
 
+
+    /**
+     * Get the tags that should be assigned to the job.
+     *
+     * @return array
+     */
+    public function tags()
+    {
+        return ['turbovote'];
```

Comment:
Thinking we should add some sort of date to the tag for better 👀 but this is just a default for now.

Reply:
@sbsmith86 what is this used for?

Reply:
@katiecrane it allows us to "tag" queued jobs so if you wanted some unique identifier for a job put on a queue you would configure that here. 

Like we can tag these jobs as all `turbovote` jobs, a general signup import job could be `signup` or whatever. And then you can monitor the tag in Horizon. Still working out the usefulness of this, but I wanted to see it in action.

Reply:
@sbsmith86 oh cool! that definitely sounds useful!